design-toolkit/components/src/components/loading-placeholder/styles.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2025 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { tv } from '@/lib/utils';
+import type { VariantProps } from 'tailwind-variants';
+
+export type LoadingPlaceholderStyleVariants = VariantProps<
+  typeof LoadingPlaceholderStyles
+>;
+
+export const LoadingPlaceholderStylesDefaults = {
+  pulse: true,
+  shape: 'rect',
+  theme: 'dark',
+} as const;
+
+export const LoadingPlaceholderStyles = tv({
+  base: 'px-l py-s',
+  variants: {
+    pulse: {
+      true: 'animate-pulse',
+    },
+    shape: {
+      circ: 'aspect-square h-[1lh] rounded-[100%]',
```

Comment:
Please use the existing `rounded-*` classes to match the theme. I believe you want `rounded-full` and `rounded-small`

Reply:
Yea, there's a number of other token naming inconsistencies that need to be addressed with design. We've surfaced those and you'll likely see a number of updates when light mode refactors come through